src/_components/card/Card.svelte

```diff
@@ -1,4 +1,6 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
+
   import type { TThemes } from '../../_utilities/data-theme/DataTheme';
```

Comment:
Bliver data-theme stadig brugt nogen steder eller kan denne fjernes? :)

Reply:
potentielt, det er kun articlecard der ikke skal have det for nu

Reply:
Alright, så den forbliver på selve Card